server/app/controllers/member-controller.js

```diff
@@ -115,4 +115,3 @@ module.exports = {
     getMemberById,
     deleteMember
 };
```

Comment:
You deleted a line here, this will be reported by *eslint* as an error (All Linux files must have a empty line at the end of it)